client/src/util/dataframe/labelIndex.ts

```diff
@@ -53,77 +87,79 @@ class IdentityInt32Index {
     return k;
   }
 
-  // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any -- - FIXME: disabled temporarily on migrate to TS.
-  getOffset(i: any) {
+  getOffset(label: LabelType): number {
     // label to offset
-    return Number.isInteger(i) && i >= 0 && i < this.maxOffset ? i : undefined;
-  }
-
-  // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any -- - FIXME: disabled temporarily on migrate to TS.
-  getOffsets(arr: any) {
-    // labels to offsets
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any --- FIXME: disabled temporarily on migrate to TS.
-    return arr.map((i: any) => this.getOffset(i));
+    return Number.isInteger(label) && label >= 0 && label < this.maxOffset
+      ? (label as number)
+      : -1;
   }
 
-  // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types, @typescript-eslint/no-explicit-any -- - FIXME: disabled temporarily on migrate to TS.
-  getLabel(i: any) {
+  getLabel(offset: number): number | undefined {
     // offset to label
-    return Number.isInteger(i) && i >= 0 && i < this.maxOffset ? i : undefined;
+    return Number.isInteger(offset) && offset >= 0 && offset < this.maxOffset
+      ? offset
+      : undefined;
```

Comment:
does it make sense to return `-1` instead of `undefined` here to have consistency with `getOffset()` above ?

Reply:
No, it doesn't.  -1 is a _legal_ label.  

Remember:
* offsets are numeric indices.  `-1` is "no such offset"
*  labels are any string|bool|number.    `undefined` is "no such label"

This is documented in the comments for `getOffset()` and `getLabel()`

Reply:
ooh got it! Thanks for the explanation 🎉 Makes sense to me now 😄 !